android/src/main/java/com/theweflex/react/WeChatModule.java

```diff
@@ -441,6 +471,10 @@ public void onResp(BaseResp baseResp) {
         } else if (baseResp instanceof SendMessageToWX.Resp) {
             SendMessageToWX.Resp resp = (SendMessageToWX.Resp) (baseResp);
             map.putString("type", "SendMessageToWX.Resp");
+        } else if (baseResp instanceof PayResp) {
+            PayResp resp = (PayResp) (baseResp);
+            map.putString("type", "PayReq.Resp");
+            map.putString("returnkey", resp.returnKey);
```

Comment:
See [this line](https://github.com/simman/react-native-wechat/blob/213ba2010a2c12df944b04bc4a4f6dd16e2058bb/index.js#L204), it causes `resp.returnKey` to be undefined :)